api/src/test/java/org/pmiops/workbench/api/ConceptSetsControllerTest.java

```diff
@@ -773,6 +785,17 @@ private void stubGetWorkspace(String ns, String name, String creator, WorkspaceA
     fcResponse.setWorkspace(fcWorkspace);
     fcResponse.setAccessLevel(access.toString());
     when(fireCloudService.getWorkspace(ns, name)).thenReturn(fcResponse);
+    stubGetWorkspaceAcl(ns, name, creator, access);
+  }
+
+  private void stubGetWorkspaceAcl(String ns, String name, String creator, WorkspaceAccessLevel access) {
```

Comment:
If you need this in several places, then maybe build a reusable `testAclImpl` or something that you can have in all these test classes.

Reply:
https://precisionmedicineinitiative.atlassian.net/secure/RapidBoard.jspa?rapidView=30&projectKey=RW&view=planning&selectedIssue=RW-2480&epics=visible&selectedEpic=RW-1801